### PR TITLE
tudu: update to 0.10.3

### PR DESCRIPTION
**Pull request by bertronika**
Updated to newest version and changed description to be more precise and
meet xlint's recommendations.

### CODE REVIEW

**Comment by bertronika** [comment]
It looks like Travis is trying to build the wrong commit (the one before closing the pull request). The newest commit now has the right modifications. Do I still have to change something or does Travis need manual intervention?

**Comment by zdtcd** [comment]
@bertronika remake a git-commit, force-push, and Travis will pick this up.

File: srcpkgs/tudu/template [hunk 1]
```diff
@@ -1,13 +1,13 @@
 # Template file for 'tudu'
 pkgname=tudu
-version=0.10.2
+version=0.10.3
 revision=1
 build_style=gnu-configure
 conf_files="/etc/tudurc"
 makedepends="ncurses-devel"
-short_desc="A command line interface to manage hierarchical todos"
+short_desc="Ncurses-based terminal user interface to manage hierarchical todos"
```
Comment (zdtcd) [inline srcpkgs/tudu/template]: I think short_desc is better to be 'Curse based tool to ....'

**Comment by bertronika** [comment]
It worked this time. Thanks, @zdtcd.

**Pull request merged**